subsys/debug/Kconfig

```diff
@@ -3,6 +3,8 @@
 # Copyright (c) 2015 Wind River Systems, Inc.
```

Comment:
split the changes in this file into 2 parts, generic tracing backend, and CTF related changes.